e2e/tests/conceptsets/copy-to-workspace.spec.ts

```diff
@@ -98,11 +102,15 @@ describe('Copy Concept Set to another workspace', () => {
   test('Workspace OWNER cannot copy Concept Set when CDR Versions mismatch', async () => {
     // Create a source and a destination workspace with differing CDR Versions.
 
-    const destWorkspaceCard: WorkspaceCard = await createWorkspace(page, config.defaultCdrVersionName);
-    const destWorkspace = await destWorkspaceCard.getWorkspaceName();
+    const [, destWorkspace] = await createWorkspace(page, {
+      cdrVersion: config.defaultCdrVersionName,
```

Comment:
`cdrVersion` default value is `config.defaultCdrVersionName` in `createWorkspace` function. It can be skipped.